packages/next/server/web/spec-extension/cookies.ts

```diff
@@ -59,10 +64,23 @@ export class NextCookies extends Cookies {
     super(response.headers.get('cookie'))
     this.response = response
   }
+  get = (...args: Parameters<Cookies['get']>) => {
```

Comment:
https://github.com/vercel/next.js/pull/36943#pullrequestreview-976847378

> Can we add explicit return type for `get`?

```suggestion
  get = (...args: Parameters<Cookies['get']>): string | undefined => {
```